docs/api.md

```diff
@@ -2316,15 +2316,6 @@ Returns SMS status for the current user.
   is in the specified country.
   <!--end-query-param-get-smsstatus-country-->
 
-##### Error responses
```

Comment:
O_o

Reply:
yeah, this happened to me as well: https://github.com/mozilla/fxa-auth-server/pull/2278/files#r165863399

Reply:
How did that get removed?

I'll put it back.